docs/js-core-2/week-2/homework.md

```diff
@@ -4,6 +4,32 @@ title: Coursework
 sidebar_label: Coursework
 ---
 
+## 0) Flipped Classroom Practice (3h)
+
+1.1 Given the following array of objects
+
+```js
+let people = [
```

Comment:
I think this array should have at least 5 objects, so it's clearer to see the effects of calling an array method on `people`